core/templates/domain/objects/ratio.model.ts

```diff
@@ -92,7 +92,7 @@ export class Ratio {
    * Returns this Ratio in its most simplified form.
    */
   convertToSimplestForm(): number[] {
-    var gcd = (x: number, y: number) => {
+    var gcd: (x: number, y: number) => number = (x: number, y: number) => {
```

Comment:
Maybe create a separate type for readability?

Reply:
done